lib/nfs_common.pm

```diff
@@ -277,9 +277,16 @@ sub check_y2_nfs_func {
     stop_service();
     # we need to cleanup the nfs settings after service check was done.
     if ($stage eq 'after') {
-        zypper_call 'rm yast2-nfs-server nfs-kernel-server', timeout => 480;
-        script_run ':> /etc/exports';
+        zypper_call 'rm yast2-nfs-server', timeout => 480;
```

Comment:
Why need remove the yast2-nfs-server? Will this affect later test?

Reply:
this is the nfs configure yast2 GUI, do not need it anymore.